pkg/api/helpers.go

```diff
@@ -449,9 +449,9 @@ func GetAffinityFromPodAnnotations(annotations map[string]string) (*Affinity, er
 	return nil, nil
 }
 
-// GetTolerationsFromPodAnnotations gets the json serialized tolerations data from Pod.Annotations
+// GetTolerationsFromAnnotations gets the json serialized tolerations data from Annotations
 // and converts it to the []Toleration type in api.
-func GetTolerationsFromPodAnnotations(annotations map[string]string) ([]Toleration, error) {
+func GetTolerationsFromAnnotations(annotations map[string]string) ([]Toleration, error) {
```

Comment:
You are renaming exported symbol. Are you sure the symbol is not used anywhere else? Even in projects importing ``k8s.io/kubernetes/pkg/api`` package?